lib/model/payload/electrocardiogram.dart

```diff
@@ -58,6 +58,65 @@ class Electrocardiogram extends Sample<ElectrocardiogramHarmonized> {
             json, ElectrocardiogramHarmonized.fromJson(json['harmonized']));
 }
 
+enum ElectrocardiogramClassification {
+  notSet,
+  sinusRhythm,
+  atrialFibrillation,
+  inconclusiveLowHeartRate,
+  inconclusiveHighHeartRate,
+  inconclusivePoorReading,
+  inconclusiveOther,
+  unrecognized,
+}
+
+extension ElectrocardiogramClassificationRawValue
+    on ElectrocardiogramClassification {
+  static ElectrocardiogramClassification parse(String data) {
+    switch (data) {
+      case 'notSet':
+        return ElectrocardiogramClassification.notSet;
+      case 'sinusRhythm':
+        return ElectrocardiogramClassification.sinusRhythm;
+      case 'atrialFibrillation':
+        return ElectrocardiogramClassification.atrialFibrillation;
+      case 'inconclusiveLowHeartRate':
+        return ElectrocardiogramClassification.inconclusiveLowHeartRate;
+      case 'inconclusiveHighHeartRate':
+        return ElectrocardiogramClassification.inconclusiveHighHeartRate;
+      case 'inconclusivePoorReading':
+        return ElectrocardiogramClassification.inconclusivePoorReading;
+      case 'inconclusiveOther':
+        return ElectrocardiogramClassification.inconclusiveOther;
+      case 'unrecognized':
+        return ElectrocardiogramClassification.unrecognized;
+      default:
+        throw ArgumentError.value('Invalid classification type');
+    }
+  }
+}
+
+enum ElectrocardiogramSymptomStatus {
+  notSet,
+  none,
+  present,
+}
+
+extension ElectrocardiogramSymptomStatusRawValue
+    on ElectrocardiogramSymptomStatus {
+  static ElectrocardiogramSymptomStatus parse(String data) {
+    switch (data) {
+      case 'notSet':
+        return ElectrocardiogramSymptomStatus.notSet;
+      case 'none':
+        return ElectrocardiogramSymptomStatus.none;
+      case 'present':
+        return ElectrocardiogramSymptomStatus.present;
+      default:
+        throw ArgumentError.value('Invalid symptom status');
+    }
+  }
+}
+
```

Comment:
I don't see the point of creating of these models, because the ECG can be generated only by Watch, so the data can be only read. By creating these enums with constructors you try to introduce and possibility to create the ECG objects here